modules/seqexec/server/src/test/scala/seqexec/server/SeqexecServerLensesSpec.scala

```diff
@@ -13,19 +12,18 @@ import seqexec.server.SeqexecServerArbitraries._
 import gem.arb.ArbObservation
 import gem.Observation
 import monocle.law.discipline.LensTests
-import seqexec.engine.{Action, Actions, Result}
+import seqexec.engine.{Action, Actions}
 
 /**
   * Tests SeqexecServer Lenses
   */
 final class SeqexecServerLensesSpec extends CatsSuite with ArbObservation {
 
   // I tried to go down the rabbit hole with the Eqs, but it is not worth it for what they are used.
-  implicit val streamEq: Eq[fs2.Stream[IO, Result]] = Eq.fromUniversalEquals
   implicit val actStateEq: Eq[Action.State] = Eq.fromUniversalEquals
-  implicit val actionEq: Eq[Action[IO]] = Eq.by(x => (x.kind, x.gen, x.state))
-  implicit val steppEq: Eq[HeaderExtraData => List[Actions[IO]]] = Eq.fromUniversalEquals
-  implicit val stepActionsGenEq: Eq[StepActionsGen] = Eq.by(x => (x.pre))
+  implicit def actionEq[F[_]]: Eq[Action[F]] = Eq.by(x => (x.kind, x.state))
```

Comment:
Maybe these shouldn't be `implicit` and pass them eplicitly to the tests instead